problem_builder/tests/integration/test_step_builder.py

```diff
@@ -765,3 +774,14 @@ def test_plot(self):
             step_builder, plot_controls, hidden=False,
             labels=['Custom Q1 label', 'Custom Q2 label', 'Custom Q3 label', 'Custom Q4 label']
         )
+
+    def test_instruction_message(self):
+        step_builder, controls = self.load_assessment_scenario("step_builder_instruction.xml", {})
+        # Step 1
+        # Submit free-form answer, go to next step
+        self.freeform_answer(None, step_builder, controls, 'This is the answer', CORRECT, hold=True, last=True)
+        message = step_builder.find_element_by_css_selector('.sb-step-message')
+        self.wait_until_visible(message)
+        self.assertEqual(message.text, 'Hello!')
+        controls.review.click()
```

Comment:
@itsjeyd I'm going to make it do just the document click, because it should work no matter where you click on the document, and clicking on a next button would be adding unneeded variables for the test.